unison-cli/src/Unison/Codebase/Editor/Output.hs

```diff
@@ -410,14 +408,6 @@ data CreatedProjectBranchFrom
   | CreatedProjectBranchFrom'OtherBranch (ProjectAndBranch Sqlite.Project Sqlite.ProjectBranch)
   | CreatedProjectBranchFrom'ParentBranch ProjectBranchName
 
-data DisplayDefinitionsOutput = DisplayDefinitionsOutput
```

Comment:
This output logic is much simpler since we now render the code in the command itself.